docs/contributing/packages.md

```diff
@@ -1,41 +1,20 @@
 # Package Updates
 
-We actively seek pull requests which update packages already included in the project Dockerfiles.
-This is a great way for first-time contributors to participate in developing the Jupyter Docker
-Stacks.
+As a general rule, we do not pin package versions in our `Dockerfile`s.
+The dependencies resolution is a difficult thing to do
+This means that packages might have old versions.
+Images are rebuilt weekly, so usually, packages receive updates quite frequently.
 
-Please follow the process below to update a package version:
+_Note: We pin major.minor version of python, so it will stay the same even after `mamba update` command._
```

Comment:
````suggestion
```{note}
We pin major.minor version of python, so it will stay the same even after `mamba update` command.
```
````